ambry-account/src/main/java/com/github/ambry/account/mysql/AccountDao.java

```diff
@@ -322,6 +337,20 @@ private void bindContainer(PreparedStatement statement, int accountId, Container
     }
   }
 
+  /**
+   * Close a resource without throwing exception.
+   * @param resource the resource to close.
+   */
+  private static void closeQuietly(AutoCloseable resource) {
```

Comment:
Let's reuse the method in SqlDataAccessor.  Better yet, can move it to MySqlUtils and make public.